namer/comparison_results.py

```diff
@@ -280,7 +280,7 @@ class ComparisonResults:
 
     def get_match(self) -> Optional[ComparisonResult]:
         match = None
-        if self.results and len(self.results) > 0 and self.results[0].is_match():
+        if self.results and self.results[0].is_match():
```

Comment:
if results are empty, wont this result in an error?

Reply:
no, `if self.results` checks if not None and if not empty